voidcrew/code/modules/overmap/ships/pvp/misc.dm

```diff
@@ -0,0 +1,55 @@
+#define SHIP_GRACE_TIME (1 MINUTES)
+
+/obj/structure/overmap/ship/simulated
+	var/obj/machinery/computer/helm/most_recent_helm // For sending messages to the target's helm
+	var/engine_cooldown = 0 // The point in time when your engines will work again
+	var/antivirus_nodes = 0
+	var/sub_grace = 0
+
+/obj/machinery/computer/helm/connect_to_shuttle(obj/docking_port/mobile/port, obj/docking_port/stationary/dock)
+	..()
+	current_ship.most_recent_helm = src
```

Comment:
this proc already exists in `voidcrew/code/modules/overmap/helm.dm` you should just put it there